js/dataset/Dataset.svelte

```diff
@@ -125,124 +125,126 @@
 	// Saw this when rendering examples in a gr.render block
 	$: selected_samples_json = JSON.stringify(selected_samples || []);
 
-	$: get_component_meta(selected_samples_json);
+	// $: get_component_meta(selected_samples_json);
 </script>
 
-{#if gallery}
-	<div class="gallery">
-		{#each selected_samples as sample_row, i}
-			{#if sample_row[0] != null}
-				<button
-					class="gallery-item"
-					on:click={() => {
-						value = i + page * samples_per_page;
-						onclick({ index: value, value: sample_row });
-						onselect({ index: value, value: sample_row });
-					}}
-					on:mouseenter={() => handle_mouseenter(i)}
-					on:mouseleave={() => handle_mouseleave()}
-				>
-					{#if sample_labels}
-						<BaseExample
-							value={sample_row[0]}
-							selected={current_hover === i}
-							type="gallery"
-						/>
-					{:else if component_meta.length}
-						{#await component_meta[0][0].component then component}
-							{#key sample_row[0]}
-								<svelte:component
-									this={component.default}
-									{...component_props[0]}
-									value={sample_row[0]}
-									{samples_dir}
-									type="gallery"
-									selected={current_hover === i}
-									index={i}
-									{root}
-								/>
-							{/key}
-						{/await}
-					{/if}
-				</button>
-			{/if}
-		{/each}
-	</div>
-{:else if selected_samples.length > 0}
-	<div class="table-wrap">
-		<table tabindex="0" role="grid">
-			<thead>
-				<tr class="tr-head">
-					{#each headers as header}
-						<th>
-							{header}
-						</th>
-					{/each}
-				</tr>
-			</thead>
-			<tbody>
-				{#each component_meta as sample_row, i}
-					<tr
-						class="tr-body"
+{#await get_component_meta(selected_samples_json) then _}
```

Comment:
On a reload, it's possible the promises don't finish resolving by the time we render the ui. Not quite sure why but it results in the Examples going missing